### ISSUE TITLE
Cant get it to log notices etc. without going to the screen

### ISSUE THREAD

**Issue by julesbl** [stevethomas/codeigniter-monolog#1]
Hi

I have been trying to get this system to log warnings, notices etc without sending to the stdout

I have error_reporting(E_ALL); in index.php

If I set display_errors = off, nothing is logged except fatal php errors
If I set log_errors = on, errors start going to stdout again

If I set error_reporting(0) nothing is logged

Any ideas?

Jules

**Comment by stevethomas** [comment]
Hi Jules,

In application/config did you increase the threshold to > 1?

**Comment by julesbl** [comment]
Hi Steve

Thanks for getting back to me.

I have tried a number of things now and have characterised the problem.

The set display_error etc has no effect on this, that was just me getting in a mess.
I have tried setting the threshold > 1 with no result.
If I set error_reporting to 0 in the index.php no php warnings or notices are logged, if I revert to the original logging system they are logged.
If I set error_reporting(E_ALL) in index.php warnings and notices are logged both in the the original system and in monolog.

Jules
